indico/modules/rb_new/client/js/common/rooms/actions.js

```diff
@@ -105,4 +105,4 @@ export function fetchAttributes(id) {
 }
 
 export const openRoomDetails = (roomId) => modalActions.openModal('room-details', roomId);
-export const openRoomDetailsBook = (roomId) => modalActions.openModal('room-details-book', roomId);
+export const openRoomDetailsBook = (roomId) => modalActions.openModal('room-details-edit', roomId);
```

Comment:
Maybe I'm missing something, but `openRoomDetailsBook` should stay, right? And you should add an `openRoomDetailsEdit`, or something like that?

Reply:
Mistake during merge and I hadn't checked that yet. Thanks for pointing out as well.